pkgs/applications/misc/buku/default.nix

```diff
@@ -40,27 +55,16 @@ with python3'.pkgs; buildPythonApplication rec {
     flake8
     pyyaml
     mypy-extensions
+    click
```

Comment:
pytest-cov, pylint and flake8 should be removed. They only check cosmetic stuff and often break for no good reason.

Reply:
555111f1a90a3519856aaf6403a0118d6ae55dee